lua/terrortown/lang/en.lua

```diff
@@ -2281,7 +2281,7 @@ L.label_execute_command = "Execute command"
 -- 2024-05-22
 L.tip38 = "You can pick up focused weapons by pressing {usekey}. It will automatically drop you blocking weapon."
 L.tip39 = "You can change your key bindings in the bindings menu, located in the Settings menu opened with {helpkey}."
-L.tip40 = "On the left side of your screen are icons showing current items or status effects applied to you."
+L.tip40 = "On the left side of your screen are icons showing current equipment or status effects applied to you."
```

Comment:
in this case this change might be misleading. On the left in the sidebar are only items and status effects, not any generic equipment

Reply:
Radar, Speedrun, Body Armor and Disguiser are all equipment (or equipment items is you want to spell it this way), isn't?

Or I am missing something?